.gitignore

```diff
@@ -11,3 +11,6 @@ npm-debug.log
 # coverage
 .nyc_output
 coverage
+
+# osx
+.DS_Store
```

Comment:
i sort of feel like it's the project's job to prevent stuff like this from getting committed, but i don't really care much either way 😄. let's see what others think